package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "simple-icons",
+  "version": "1.0.0",
+  "description": "SVG icons for popular brands https://simpleicons.org",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"no tests available\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/danleech/simple-icons.git"
+  },
+  "keywords": [
+    "brand",
+    "branding",
+    "brands",
+    "brand-assets",
+    "icons",
+    "logos",
+    "svg",
+    "svg-icons",
+    "design",
+    "design-assets"
+  ],
+  "author": "Dan Leech <dan@danleech.com> (https://danleech.com)",
+  "license": "SEE LICENSE IN LICENSE.md",
```

Comment:
Since the license type used for this package is pretty common, the value for `license` can be set to `	CC0-1.0`. For more info read [this](https://docs.npmjs.com/files/package.json#license) 🙂